Source/Core/Core/IOS/ES/ES.cpp

```diff
@@ -911,19 +996,14 @@ IPCCommandResult ES::GetTitles(const IOCtlVRequest& request)
   if (!request.HasNumberOfValidVectors(1, 1))
     return GetDefaultReply(ES_PARAMETER_SIZE_OR_ALIGNMENT);
 
-  u32 MaxCount = Memory::Read_U32(request.in_vectors[0].address);
-  u32 Count = 0;
-  for (int i = 0; i < (int)s_title_ids.size(); i++)
+  const std::vector<u64> titles = GetInstalledTitles();
```

Comment:
The below code seems to be almost the same in GetOwnedTitles() as the one here, with the only thing differing being where the vector comes from. Would it be feasible to extract it to a utility function that takes a `std::vector` and an `IOCtlVRequest` (or simply the address directly), or is it not worth it?